doc/tutorials/spark_estimator.rst

```diff
@@ -51,17 +63,141 @@ type or spark array type.
 The above snippet code returns a `transformed_test_spark_dataframe` that contains the input
 dataset columns and an appended column "prediction" representing the prediction results.
 
-
-******************
 SparkXGBClassifier
-******************
-
+==================
 
 `SparkXGBClassifier` estimator has similar API with `SparkXGBRegressor`, but it has some
 pyspark classifier specific params, e.g. `raw_prediction_col` and `probability_col` parameters.
 Correspondingly, by default, `SparkXGBClassifierModel` transforming test dataset will
 generate result dataset with 3 new columns:
-
 - "prediction": represents the predicted label.
 - "raw_prediction": represents the output margin values.
 - "probability": represents the prediction probability on each label.
+
+
+***************************
+XGBoost PySpark GPU support
+***************************
+
+XGBoost PySpark supports GPU training and prediction. To enable GPU support, you first need
+to install the xgboost and cudf packages. Then you can set `use_gpu` parameter to `True`.
+
+Below tutorial will show you how to train a model with XGBoost PySpark GPU on Spark
+standalone cluster.
+
+
+Write your PySpark application
+==============================
+
+.. code-block:: python
+
+  from xgboost.spark import SparkXGBRegressor
+  spark = SparkSession.builder.getOrCreate()
+
+  # read data into spark dataframe
+  train_data_path = "xxxx/train"
+  train_df = spark.read.parquet(data_path)
+
+  test_data_path = "xxxx/test"
+  test_df = spark.read.parquet(test_data_path)
+
+  # assume the label column is named "class"
+  label_name = "class"
+
+  # get a list with feature column names
+  feature_names = [x.name for x in train_df.schema if x.name != label]
+
+  # create a xgboost pyspark regressor estimator and set use_gpu=True
+  regressor = SparkXGBRegressor(
+    features_col=feature_names,
+    label_col=label_name,
+    num_workers=2,
+    use_gpu=True,
+  )
+
+  # train and return the model
+  model = regressor.fit(train_df)
+
+  # predict on test data
+  predict_df = model.transform(test_df)
+  predict_df.show()
+
+Prepare the necessary packages
+==============================
+
+We recommend using Conda or Virtualenv to manage python dependencies
+in PySpark. Please refer to
+`How to Manage Python Dependencies in PySpark <https://www.databricks.com/blog/2020/12/22/how-to-manage-python-dependencies-in-pyspark.html>`_.
```

Comment:
This varies between different providers of pyspark environments. On dataproc we can't submit the environment through spark-submit.

Reply:
I mentioned this tutorial is for spark standalone mode, I didn't want to involve other CSPs to xgboost.